enableRounds.toml

```diff
@@ -10,4 +10,4 @@
 [RoundActivations]
     [RoundActivations.DisableAsyncCallV1]
         Options = []
-        Round = "18216765" # epoch 1265 should begin at round 14401 * 1265 = 18217265 (500 will be subtracted)
+        Round = "100"
```

Comment:
Can't we use `0`? And remove the `TODOs` from `WasmVMVersions`?

Reply:
Right